develop/blockchains/get-started/build-custom-blockchains.md

```diff
@@ -0,0 +1,58 @@
+
+- [Pop Network](https://learn.onpop.io/appchains/pop-cli/new#templates){target=\_blank} - designed with user-friendliness in mind, Pop Network offers an approachable starting point for new developers, with a simple CLI interface for creating appchains 
+
+Choosing the suitable template depends on your project’s unique requirements, level of customization, and integration needs. Starting from a template speeds up development and enables you to focus on implementing your chain’s unique features rather than the foundational blockchain setup.
+
+## High-Level Steps to Build a Custom Chain
+
+To build a custom blockchain using the Polkadot SDK, there exist some general steps to follow:
+
+Building a custom blockchain with the Polkadot SDK involves several core steps, from environment setup to deployment. Here’s a breakdown of each stage:
+
+- **Set up the development environment** - install Rust and configure all necessary dependencies to work with the Polkadot SDK (for more information, check the [Install Polkadot SDK dependencies](TODO:update-path){target=\_blank} page). Ensuring your environment is correctly set up from the start is crucial for avoiding compatibility issues later
+
+- **Clone the chain template** - start by downloading the code for one of the pre-built templates that best aligns with your project needs. Each template offers a different configuration, so select one based on your chain’s intended functionality and scalability requirements
+
+- **Define your chain's custom logic** - with your chosen template, check the runtime configuration to customize the chain’s functionality. Polkadot’s modular “pallet” system lets you to easily add or modify features like account balances, transaction handling, and staking. Creating custom pallets to implement unique features and combining them with existing ones enables you to define the unique aspects of your chain
+
+- **Test and debug** - testing is essential to ensure your custom chain works as intended. Conduct unit tests for individual pallets and integration tests for interactions between pallets
+
+- **Compile** - after finalizing and testing your custom configurations, compile the blockchain to generate the necessary executable files for running a node. Run the node locally to validate that your customizations work as expected and that your chain is stable and responsive
+
+- **Deploy to a testnet or mainnet** - once testing is complete, deploy your chain to a live environment. Depending on your project’s goals, you can start with a testnet to monitor performance and gather feedback or launch directly on a mainnet
+
+Each of these steps is designed to build on the last, helping ensure that your custom blockchain is functional, optimized, and ready for deployment within the Polkadot ecosystem or beyond.
+
+## Monitoring and Maintenance
```

Comment:
Link the monitoring / "General Management" page we have maybe?

Reply:
great! added on [b34fec3](https://github.com/polkadot-developers/polkadot-docs/pull/156/commits/b34fec302747815381ed88074cae9b0ff5f4f73c), thank you!